fem/libceed/ceed.cpp

```diff
@@ -402,7 +404,7 @@ void CeedPAAssemble(const CeedPAOperator& op,
    CeedQFunctionSetContext(ceedData.apply_qfunc, &ceedData.build_ctx,
                            sizeof(ceedData.build_ctx));
```

Comment:
The QFunctionContext interface needs to be updated here too (see #1701 and #1705).